saga/src/main/java/io/axoniq/dev/samples/saga/ProcessOrderSaga.java

```diff
@@ -51,7 +54,7 @@
 
     @StartSaga
     @SagaEventHandler(associationProperty = ORDER_ID_ASSOCIATION)
-    public void on(OrderConfirmedEvent event, DeadlineManager deadlineManager, UUIDProvider uuidProvider) {
+    public void on(OrderConfirmedEvent event, DeadlineManager deadlineManager, UUIDProvider uuidProvider, EventScheduler axonServerEventScheduler) {
```

Comment:
I think we can simply use the `EventScheduler`, instead of the `AxonServerEventScheduler`.
Thus, an import statement can be removed, as well as renaming this parameter.